initialize/initialize.go

```diff
@@ -7,7 +7,8 @@ import (
 	"log"
 	"os"
 	"sort"
-	"strings"
+  "strings"
+	"net/url"
```

Comment:
needs a `go fmt`

Reply:
Oops. That'll teach me to edit stuff directly in GitHub's editor. It converted the tab to spaces w
en I fixed the merge conflict. I'll fix it up.